frontend/utils.tsx

```diff
@@ -18,14 +18,18 @@ export function stripTags(val: string): string {
     .trim()
 }
 
-export function useSetting(key: string): string | boolean | null {
-  switch (key) {
-    case 'FAIR_OPEN':
-      return false
-    case 'FAIR_REGISTRATION_OPEN':
-      return false
+export const OptionsContext = createContext({})
+
+export function useSetting(key: string): string | number | boolean | null {
+  const options = useContext(OptionsContext)
+  const value = options[key] ?? null
+  if (value === 'true') {
+    return true
+  }
+  if (value === 'false') {
+    return false
```

Comment:
@davish I specify that these are the boolean type on the server, is there any reason why they aren't casted in the JSON response?

I'd be happy to change this in the library if it doesn't break anything else.